apps/wallet/src/ui/app/shared/Portal.tsx

```diff
@@ -0,0 +1,24 @@
+// Copyright (c) Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+import { useEffect, useState } from 'react';
+import { createPortal } from 'react-dom';
+
+type PortalProps = {
+    children: React.ReactNode;
+    containerId: string;
```

Comment:
Why not just create a new DOM node on render instead of forcing a container ID?

Reply:
ahh - that slipped in from another branch! will remove 

@williamrobertson13 some feedback for your Portal PR :)

Reply:
I think that might be a matter of preference - i.e. should the component attach to an existing node in the DOM, or should it modify the DOM and attach itself? Feel free to comment on the PR I closed earlier if there's a strong preference to do this the other way (would love to hear your reasoning)

Reply:
Yeah I generally prefer my portals to create DOM nodes so that they don't depend on external DOM structure (the only DOM structure I want to be aware of is the single node that I mount into at the application root).